Makefile

```diff
@@ -4,6 +4,9 @@ build-volumes:
 	@ sudo mkdir -p  ./airflow2/logs ./airflow2/plugins ./airflow2/dags
 	@ sudo chmod 777 ./airflow2/logs ./airflow2/plugins ./airflow2/dags
 
+build-network:
+	@ docker network create -d bridge elk
```

Comment:
Why the default network is not sufficient? 
Do we want to use this network across docker-container files? And if so then perhaps we can call it something else.